istioctl/cmd/analyze.go

```diff
@@ -118,7 +130,7 @@ istioctl experimental analyze -k -d false
 			}
 
 			for _, m := range messages {
-				fmt.Printf("%v\n", m.String())
+				fmt.Printf("%s%v%s\n", colorPrefix(m), m.String(), colorSuffix())
```

Comment:
Should we just colorize the level (Info/Warn/Error) instead of the whole message? That is how some logging system do things

Reply:
OK, now it only colorizes the level.  The red and green text was hard to read with some console color schemes.